acceptance/analyzer_test.go

```diff
@@ -89,11 +102,88 @@ func TestAnalyzer(t *testing.T) {
 	)
 	defer h.DockerImageRemove(t, analyzeImage)
 
+	fixtures = setupAnalyzeFixtures(t)
+	defer fixtures.removeAll(t)
+
 	for _, platformAPI := range api.Platform.Supported {
 		spec.Run(t, "acceptance-analyzer/"+platformAPI.String(), testAnalyzerFunc(platformAPI.String()), spec.Parallel(), spec.Report(report.Terminal{}))
 	}
 }
 
+func setupAnalyzeFixtures(t *testing.T) analyzeFixtures {
+	var fixtures analyzeFixtures
+
+	appMeta := minifyMetadata(t, filepath.Join("testdata", "analyzer", "app_image_metadata.json"), platform.LayersMetadata{})
+	cacheMeta := minifyMetadata(t, filepath.Join("testdata", "analyzer", "cache_image_metadata.json"), platform.CacheMetadata{})
+
+	// Registry
+
+	regAppName := "some-app-image-" + h.RandString(10)
+	fixtures.authRegAppImage, fixtures.authRegAuthConfig = buildAuthRegistryImage(
+		t,
+		regAppName,
+		filepath.Join("testdata", "analyzer", "app-image"),
+		"--build-arg", "fromImage="+containerBaseImage,
+		"--build-arg", "metadata="+appMeta,
+	)
+	fixtures.readOnlyRegAppImage = noAuthRegistry.RepoName(regAppName)
+
+	regOtherAppName := "some-other-app-image-" + h.RandString(10)
+	fixtures.authRegOtherAppImage, _ = buildAuthRegistryImage(
+		t,
+		regOtherAppName,
+		filepath.Join("testdata", "analyzer", "app-image"),
+		"--build-arg", "fromImage="+containerBaseImage,
+		"--build-arg", "metadata="+appMeta,
+	)
+
+	regCacheName := "some-cache-image-" + h.RandString(10)
+	fixtures.authRegCacheImage, _ = buildAuthRegistryImage(
+		t,
+		regCacheName,
+		filepath.Join("testdata", "analyzer", "cache-image"),
+		"--build-arg", "fromImage="+containerBaseImage,
+		"--build-arg", "metadata="+cacheMeta,
+	)
+	fixtures.readOnlyRegCacheImage = noAuthRegistry.RepoName(regCacheName)
+
+	// Daemon
+
+	fixtures.daemonAppImage = "some-app-image-" + h.RandString(10)
+	cmd := exec.Command(
+		"docker",
+		"build",
+		"-t", fixtures.daemonAppImage,
+		"--build-arg", "fromImage="+containerBaseImage,
+		"--build-arg", "metadata="+appMeta,
+		filepath.Join("testdata", "analyzer", "app-image"),
+	) // #nosec G204
+	h.Run(t, cmd)
+
+	fixtures.daemonCacheImage = "some-cache-image-" + h.RandString(10)
```

Comment:
```suggestion
	fixtures.daemonCacheImage = regCacheName
```
Same comment about renaming as [here](https://github.com/buildpacks/lifecycle/pull/668/files#r674153652).